mc-react-components/src/lib/map-filter-container/polygon-selection-ui.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { PolygonSelectionUi } from './polygon-selection-ui';
+import { DrawType } from '../models';
+import { MenuItem, Menu, Button } from '@material-ui/core';
+
+const startDraw = jest.fn();
+const cancelDraw = jest.fn();
+const resetDraw = jest.fn();
+afterEach(() => {
+  startDraw.mockClear();
+  cancelDraw.mockClear();
+  resetDraw.mockClear();
+});
+
+// it('renders correctly', () => {
+//   // const store = rootStore.create({}, { fetch: conflictFetcher })
+//   const tree = renderer
+//     .create(<PolygonSelectionUi />)
+//     .toJSON();
+//   expect(tree).toMatchSnapshot();
+// });
```

Comment:
for now, let's create a subdirectory `__snapshots__` for each spec folder.

Reply:
will be fixed in https://github.com/MapColonies/conflicts-ui/issues/9.